### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by parvez-rashid**
Created a simple cart component.

### CODE REVIEW

**Review by Bibin-Antony — APPROVED** [review]
seems ok 👍🏻

File: index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    <link rel="stylesheet" href="/styles/cart.css" />
+    <title>Your cart | Sanguine Store</title>
+  </head>
+  <body>
+    <nav class="navigation">
+      <div class="nav-logo">SANGUINE store</div>
+      <label for="search" class="text-size-sm">
+        <input
+          type="search"
+          class="search-box padding-md border-rad-md"
```
Comment (malaykhakhar) [inline index.html:20]: Good use of utility classes.

**Review by malaykhakhar — COMMENTED** [review]
A UI question: What if you add one more item to the cart? Will the height of the checkout section increase along with the cart-items section?
If that happens you might want to fix the max height of the checkout section.

**Comment by parvez-rashid** [comment]
> A UI question: What if you add one more item to the cart? Will the height of the checkout section increase along with the cart-items section? If that happens you might want to fix the max height of the checkout section.

Thanks for pointing that out, I'll try to fix this.

**Pull request merged**